app/src/main/java/org/simple/clinic/util/UuidTypeConverters.kt

```diff
@@ -18,6 +18,32 @@ class UuidRoomTypeConverter {
   }
 }
 
+class UuidListRoomTypeConverter {
+
+  private val uuidConverter = UuidRoomTypeConverter()
+
+  @TypeConverter
+  fun toUuids(value: String?): List<UUID>? {
+    if (value == null) {
+      return null
+    }
+
+    return value.split(delimiters = *arrayOf(","))
+        .map { uuidConverter.toUuid(it)!! }
+        .toList()
+  }
+
+  @TypeConverter
+  fun fromUuids(uuids: List<UUID>?): String? {
+    if (uuids == null) {
+      return null
+    }
+    return uuids
+        .map { uuidConverter.fromUuid(it) }
+        .joinToString(separator = ",")
```

Comment:
Check out the `transform` parameter in the `joinToString` function since we might be able to skip the `map` call before this.

Reply:
Done